master/job.go

```diff
@@ -0,0 +1,164 @@
+package master
+
+import (
+	"bytes"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+// NOTE: consider using math/big for some of the operations in this package
+
+var charSetSlice [][]byte
+
+func init() {
+	nums := "0123456789"
+	alphaLower := "abcdefghijklmnopqrstuvwxyz"
+	alphaMixed := alphaLower + "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	alphaNumLower := nums + alphaLower
+	charSetSlice = [][]byte{
+		[]byte(nums),
+		[]byte(alphaLower),
+		[]byte(alphaMixed),
+		[]byte(alphaNumLower)}
+}
+
+type job struct {
+	lib.Job
+	id    int
+	Tasks []lib.Task
+}
+
+// SplitJob attempts to split a cracking job into equal sized tasks
```

Comment:
What does `size` represent as an argument? It is not clear to me what it means. For example, what does it mean if `size = 10` is passed as an argument?